apps/test/unit/templates/sectionsRefresh/SingleSectionSetUpTest.jsx

```diff
@@ -26,25 +27,41 @@ describe('SingleSectionSetUp', () => {
     const wrapper = shallow(
       <SingleSectionSetUp
         sectionNum={1}
-        section={{}}
+        section={{participantType: 'student'}}
         updateSection={() => {}}
+        isNewSection={false}
       />
     );
 
     const multiSelectGroup = wrapper.find('MultiSelectGroup');
+    expect(multiSelectGroup.length).to.equal(1);
     expect(multiSelectGroup.prop('name')).to.eq('grades');
     expect(multiSelectGroup.prop('required')).to.eq(true);
     expect(multiSelectGroup.prop('options').length).to.eq(14); // K + 12 + Other
     expect(multiSelectGroup.prop('values')).to.eql([]);
   });
 
+  it('does not render grade selector when participantType is teacher', () => {
+    const wrapper = shallow(
+      <SingleSectionSetUp
+        sectionNum={1}
+        section={{participantType: 'teacher'}}
+        updateSection={() => {}}
+        isNewSection={false}
+      />
+    );
+
+    expect(wrapper.find('MultiSelectGroup').length).to.equal(0);
+  });
+
```

Comment:
I wanted to add a test for a new section, but was having trouble writing a test with queryParams.  Do we think these tests are needed also?

Reply:
We can come back to those!